recipes/openssh/files/openssh.run

```diff
@@ -1,3 +1,3 @@
 #!/bin/execlineb -P
-
+foreground { if { test ! -e /var/log/lastlog } touch /var/log/lastlog }
```

Comment:
should this really be done in openssh? would probably end up as missing in other recipes as well then